src/directives/ng-file-drop.ts

```diff
@@ -35,7 +35,7 @@ export class NgFileDropDirective implements OnChanges, OnInit {
     this.uploader._emitter.subscribe((data: any) => {
       this.onUpload.emit(data);
       if (data.done && this.files && this.files.length) {
-        this.files = this.files.filter(f => f.name !== data.originalName);
+        this.files = Array.prototype.filter.call(this.files, (x: any) => x.name !== data.originalName)
```

Comment:
okay, here's the catch. can you please update to `[].filter.call..` and bump version in `package.json` so I can publish it to npm. Thanks.